app/javascript/pages/components/Calendar.jsx

```diff
@@ -111,25 +112,30 @@ class Calendar extends React.Component {
 
       function labels(svg) {
         let label = svg.append('g')
-          .style('font', 'bold 12px var(--sans-serif)')
+          .style('font', 'bold $font_size-base var(--sans-serif)')
```

Comment:
I'm not sure why, but somehow the labels look way smaller than the paragraph text despite both being 16px/1 rem.  Is there any sort of d3-specific relativity that might be causing this?

Reply:
Removed reference to variable